presentation/index.html

```diff
@@ -210,10 +210,10 @@
 * Get Alignak daemons status, number of hosts and services per state <!-- .element: class="fragment" data-fragment-index="4" -->
 * Be notified when problems are raised by Alignak <!-- .element: class="fragment" data-fragment-index="5" -->
 * View, in real time, an host synthesis (host and its services state) <!-- .element: class="fragment" data-fragment-index="6" -->
-* and more... <!-- .element: class="fragment" data-fragment-index="7" -->
+* Acknowledge or schedule downtimes for your hosts and services <!-- .element: class="fragment" data-fragment-index="7" -->
+* And more... <!-- .element: class="fragment" data-fragment-index="8" -->
 
-Alignak-app is a multi-screen application. It notifies you on your active screen <!-- .element: class="fragment" data-fragment-index="8" -->
-Alignak-app is compatible with Linux and Windows desktop environments (Python 2.7 / 3). <!-- .element: class="fragment" data-fragment-index="9" -->
```

Comment:
It's already said before `Alignak-app is a desktop application, in the system tray, for Linux or Windows.` line 205. That's why I've removed that part.